packages/common/src/index.ts

```diff
@@ -1112,7 +1112,7 @@ export function itemsInMetricQuery(
 }
 
 function formatRawValue(
-    field: Field | Metric | TableCalculation | CustomDimension,
+    field: Field | Metric | TableCalculation | CustomDimension | undefined,
```

Comment:
Previously `field` could already go as undefined after array access, TS just didn't infer the type as this